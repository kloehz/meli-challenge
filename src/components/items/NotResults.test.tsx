import { shallow } from "enzyme"
import { Link } from "react-router-dom";
import { NotResuts } from "./NotResuts"

describe('NotResults Component tests', () => {
    const wrapper = shallow(<NotResuts />);

    test('should be render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });

    test('should be redirect to correct path', () => {
        const link = wrapper.find(Link);
        expect(link.prop('to')).toBe('/');
    });

    test('should be show the correct text', () => {
        expect(wrapper.find('h2').text()).toBe('No se encrontraron resultados para su busqueda.');
    })
    
    
})
