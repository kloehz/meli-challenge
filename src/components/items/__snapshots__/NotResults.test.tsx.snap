// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotResults Component tests should be render correctly 1`] = `ShallowWrapper {}`;
